mcs/class/corlib/System.IO/Path.cs

```diff
@@ -284,7 +284,7 @@ public static string GetFullPath (string path)
 
 			SecurityManager.EnsureElevatedPermissions (); // this is a no-op outside moonlight
 
-#if !MOBILE
+#if WIN_PLATFORM
 			if (SecurityManager.SecurityEnabled) {
 				new FileIOPermission (FileIOPermissionAccess.PathDiscovery, fullpath).Demand ();
```

Comment:
I just renamed the define to MONO_FEATURE_CAS to be consistent with the other MONO_FEATURE_* defines.